backend/src/main/java/com/woowacourse/kkogkkog/infrastructure/SlackRequester.java

```diff
@@ -0,0 +1,91 @@
+package com.woowacourse.kkogkkog.infrastructure;
+
+import com.woowacourse.kkogkkog.exception.auth.ErrorResponseToGetAccessTokenException;
+import com.woowacourse.kkogkkog.exception.auth.UnableToGetTokenResponseException;
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.springframework.web.util.UriBuilder;
+
+@Component
+public class SlackRequester {
```

Comment:
클래스명이 좀 그렇네요. 개선방안을 찾아봅시다.
- https://www.yegor256.com/2015/03/09/objects-end-with-er.html

```suggestion
@Component
public class SlackWebClient {
```

Reply:
넵 우선 헷갈리니 그대로 나두고 내일 바꾸면 좋을 것 같습니다.